### PR TITLE
Update documentation to include environment variables

### PR DESCRIPTION
**Pull request by jtquach1**
## Describe your changes

Please include a summary of the changes and the related issue/task. Please also include relevant motivation and context. List any dependencies that are required for this change, including links to other pull requests/branches in other repositories if applicable.

This pull request lists environment variables and GitHub Secrets used in these related repositories at the following commits:
- test-ehr - https://github.com/mcode/test-ehr/commit/be33d8da2621edd137852e04cd4221706732bc2d
- crd-request-generator - https://github.com/mcode/crd-request-generator/commit/297c79733cc11d685b05e07bb26559aa48569205
- dtr - https://github.com/mcode/dtr/commit/6e4d0870daa56e00db92b7270f8db4f397b5bb2b
- REMS - https://github.com/mcode/REMS/commit/db325039e2cd5e0cc03034ada2a30f130c75060d
- pims - https://github.com/mcode/pims/commit/3661606932e4e94b32762309125187c42cfc8e78
- rems-smart-on-fhir - https://github.com/mcode/rems-smart-on-fhir/commit/23037dff5dd4856e9ec326c0e0506f02c3087475

## Issue ticket number and Jira link

Please include the Jira Ticket Number and Link for this issue/task.

https://jira.mitre.org/browse/REMS-410

## Checklist before requesting a review
- [x] I have performed a self-review of my code
- [x] Ensure the target / base branch for any feature PR is set to `dev` not main (the only exception to this is releases from `dev` and hotfix branches)

## Checklist for conducting a review
- [ ] Review the code changes and make sure they all make sense and are necessary. 
- [ ] Pull the PR branch locally and test by running through workflow and making sure everything works as it is supposed to. 

## Workflow 

Owner of the Pull Request will be responsible for merge after all requirements are met, including approval from at least one reviewer. Additional changes made after a review will dismiss any approvals and require re-review of the additional updates. Auto merging can be enabled below if additional changes are likely not to be needed. The bot will auto assign reviewers to your Pull Request for you.

### CODE REVIEW

File: EnvironmentVariables.md
Comment (zacharyrobin) [inline EnvironmentVariables.md:1]: You can remove the entire secret section, I don't think these env variables are necessary in this documentation

Comment (zacharyrobin) [inline EnvironmentVariables.md:1]: Would also include the current port numbers as current hard coded variables as they are also something that will be modifiable from the start command. Looks like you have it for REMS but would also add it for the others

Comment (zacharyrobin) [reply EnvironmentVariables.md:1]: All the `_PID` can be removed, these are docker values that are not environment variables

Comment (zacharyrobin) [reply EnvironmentVariables.md:1]: `rems-smart-on-fhir` has a few env variables that are not listed here. The names will be updated and a few more will be added with the latest merge, may need to wait on Pat's branch but would revisit. 
`REACT_APP_REMS_ADMIN_SERVER_BASE`
`REACT_APP_REMS_HOOKS_PATH`
`PORT`
(and one other that im blanking on)

**Review by zacharyrobin — CHANGES_REQUESTED** [review]

**Comment by jtquach1** [comment]
Just rebased off origin/dev at https://github.com/mcode/REMS/commit/00879b0f2a9066aafdff4ad4967a8ca1b2f90b4d, but I haven't included any environment variables that could've been potentially added from then.

File: EnvironmentVariables.md
Comment (jtquach1) [reply EnvironmentVariables.md:1]: 1. Included default values for the port numbers and hardcoded variables for repos besides REMS.
2. Removed those `_PID` values
3. Added environment variables from `rems-smart-on-fhir`'s `.env` file

**Review by zacharyrobin — APPROVED** [review]
lgtm

**Pull request merged**